src/robusta/integrations/slack/sender.py

```diff
@@ -257,6 +272,27 @@ def send_finding_to_slack(
             else:
                 blocks.extend(enrichment.blocks)
 
-        self.__send_blocks_to_slack(
+        return self.__send_blocks_to_slack(
             blocks, attachment_blocks, finding.title, slack_channel,unfurl, sink_name, finding.severity
         )
+
+    def send_welcome_message_to_slack_channel(self, channel_name) -> bool:
```

Comment:
I think it would be easier to use the slack sdk client for this purpose, and not the slack sender.
The slack sender has a lot more dependencies, so using it in the CLI will make it harder to separate later on.
In addition, the hello message isn't really a finding. wdyt?

Reply:
sounds good